src/fsharp/vs/ServiceUntypedParse.fs

```diff
@@ -988,21 +988,31 @@ module UntypedParseImpl =
                 | {TypeName = LongIdentWithDots([x], _)} when x.idText = name -> Some ()
                 | _ -> None
             
+            // This might look confusing but what's done here is quite simple
```

Comment:
Please remove this line from the comment. It gives zero information

Reply:
Thanks, you're right.